airflow/contrib/kubernetes/pod.py

```diff
@@ -120,7 +122,8 @@ def __init__(
             security_context=None,
             configmaps=None,
             pod_runtime_info_envs=None,
-            dnspolicy=None
+            dnspolicy=None,
+            prestop_wait_time="0",
```

Comment:
`"0"` vs `0` -- Won't this fail when it trickles down to `         if pod.prestop_wait_time > 0` (because of wrong types)